components/affiliates/integratedBySection.tsx

```diff
@@ -1,51 +1,166 @@
 import React, { FunctionComponent } from "react";
-import homeStyles from "../../styles/Home.module.css";
-import styles from "../../styles/affiliates.module.css";
-import { CDNImg } from "../cdn/image";
-
+import EkuboIcon from "../icons/ekubo";
+import MySwapIcon from "../icons/myswap";
+import Image from "next/image";
+import Partners from "../partners/partners";
+import { Partner } from "../partners/partners";
 const IntegratedBySection: FunctionComponent = () => {
+  const partners: Partner[] = [
```

Comment:
The partners are defined in `components/home/homeSection2.tsx` too. Can you define them once in a new `utils/partners.ts` file